packages/admin-panel/package.json

```diff
@@ -36,14 +36,13 @@
     "@tupaia/ui-chart-components": "workspace:*",
     "@tupaia/ui-components": "workspace:*",
     "@tupaia/utils": "workspace:*",
-    "ace-builds": "^1.15.3",
+    "ace-builds": "^1.10.1",
```

Comment:
**ace-builds** is all the way up to 1.32.6, but we already have it as a transitive dependency via **jsoneditor** (which uses 1.10.1) and **react-ace** (which uses 1.4.14), so I just went with the newer of the two

This seems to resolve the bug where characters get inserted multiple times when typing in the SQL editor